IdentityCore/src/intune/MSIDIntuneApplicationStateManager.h

```diff
@@ -29,7 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->